Code/Framework/AzFramework/AzFramework/Physics/Configuration/RigidBodyConfiguration.h

```diff
@@ -62,6 +62,16 @@ namespace AzPhysics
         bool m_computeInertiaTensor = true;
         bool m_computeMass = true;
 
+        //! Flags to restrict motion along specific world-space axes.
```

Comment:
nit: This likely won't come out correctly in doxygen (the comment will just be applied to the first member) you might want to check the @group annotation (or just make this a regular `//` comment) - same below

Reply:
Good point, will fix.